website/pages/docs/k8s/installation/platforms/minikube.mdx

```diff
@@ -14,3 +14,5 @@ The Learn guide includes how to:
 - Install Consul on Minikube with the official Helm chart. After
 - Access Consul with the Consul UI, CLI, and API
 - Deploy services in the Consul Connect service mesh.
+
+## Test <hi>
```

Comment:
remove? :)

Reply:
Yeah I actually need to delete all of these files because they link directly out to the learn guides now.